### DISCUSSION TITLE
Better wiki for missions options (enchanting, island...)

### DISCUSSION THREAD

**Discussion by Zeldrix** [BG-Software-LLC/SuperiorSkyblock2#881] [Approved Feature Requests] [answered]
Could you do a clearer wiki for the missions options because for example just say that we need coding knowledge to do succes-check it's pretty vague. Same for enchanting missions we dont have the list of enchantings (we have just the example). And the example for enchanting is only working with diamond.

Thank you for reading me

**Comment by OmerBenGera** [comment, marked as answer]
Will do.
About a list of enchanting, these are Bukkit enchantments which you can find on google by googling "spigot Enchantment".
